libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -159,8 +159,8 @@ AP_GPS_ERB::_parse_gps(void)
         state.location.alt    = (int32_t)(_buffer.pos.altitude_msl * 100);
         state.status          = next_fix;
         _new_position = true;
-        state.horizontal_accuracy = _buffer.pos.horizontal_accuracy * 1.0e-3f;
-        state.vertical_accuracy = _buffer.pos.vertical_accuracy * 1.0e-3f;
+        state.horizontal_accuracy = _buffer.pos.horizontal_accuracy * 1.0e-3f * 0.83f; // multiply by 0.83 to convert RMS value into CEP value
+        state.vertical_accuracy = _buffer.pos.vertical_accuracy * 1.0e-3f * 0.83f;     // multiply by 0.83 to convert RMS value into CEP value
```

Comment:
Yes, THAT is the problem our code does not mention this as well. But as you know comparing Potatos with orabges does not make any sense. Hence my quest to add unit information all over the place. OK, now back to topic... 
Emlid reach uses the open source library RTKLIB, and I know the place in the code where this calculation gets done, and it is RMS. So ... no documentation, but sorce code is available that is how I know it.

This patch adds RMS in some places were I checked the .pdfs to see what kind of values was produced.